web_source/travellers/DeborahJohnstone.qmd

```diff
@@ -0,0 +1,31 @@
+---
+title: "Deborah's traveller page"
+subtitle: "All about Deborah in this Git workshop"
+image: https://avatars.githubusercontent.com/u/135114616?v=4
+toc: false
+about:
+  id: person-profile
+  template: jolla
+---
+
+```{=html}
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item"><a href="../travellers.html">Travellers</a></li>
+    <li class="breadcrumb-item active" aria-current="page">{{< meta title >}}</li>
+  </ol>
+</nav>
+```
+
+:::{#person-profile}
+:::
+
+## Biography
+
+Vegetarian food lover, originally from Scotland and living in Essex. Mum of two girls, a senior cat and a Spanish rescue dog. 
+
+## Travelled cities
+
+- Paris
+- Barcelona
+- DUblin
```

Comment:
Amend needed for Dublin Caps has stayed on for 1st 2 letters

Reply:
I think I have fixed this and resubmitted. Hopefully it works!